kubectl-plugin/kubectl-waiops

```diff
@@ -1,4 +1,30 @@
 #!/bin/bash
+
+# The CP4WAIOps status checker tool prints out several components to indicate the status of your install.
+# In a successful install, the components below should all have a successful STATUS/PROGRESS/PHASE indicator.
+# If any of the components are incomplete or are not reconciled after a significant period of time, 
+# please refer to the IBM troubleshooting docs online or open a support case. 
+
+# Various formatting elements
+newline='\n\n'
+red=$(tput setaf 1)
+green=$(tput setaf 2)
+blue=$(tput setaf 4)
+white=$(tput setaf 7)
+bold=$(tput bold)
+normal=$(tput sgr0) #reset color/bolding
+
+export CP4WAIOPS_VERSION="3.3"
+
+# Get installation namespace
+if [ "$PROJECT_NAMESPACE" == "" ]
+then
+    printf "
+${bold}${blue}Enter your Cloud Pak for Watson AIOps installation namespace: ${normal}"
+    read PROJECT_NAMESPACE
+    oc project ${PROJECT_NAMESPACE}
+fi
+
```

Comment:
We don't require this today, this seems like a step backwards for a user experience. Could you identify?

Today, the status uses all namespaces and prints the info.